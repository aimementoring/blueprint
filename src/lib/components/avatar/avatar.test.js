import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from './avatar.js';

describe('Avatar name', () => {
  it('renders properly', () => {
    const tree = renderer.create(<Avatar text="Kevin Bardi" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Avatar with photo', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(
        <Avatar
          text="Lionel Messi"
          photo={[
            {
              url:
                'https://e00-marca.uecdn.es/assets/multimedia/imagenes/2018/08/19/15347041965884.jpg',
            },
          ]}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
