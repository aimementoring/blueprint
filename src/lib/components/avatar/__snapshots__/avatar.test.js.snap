// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar empty renders properly 1`] = `
<div
  className="theme-portal"
>
  <div
    className="profileAvatar"
  >
    <span>
      UNDEFINEDUNDEFINED
    </span>
  </div>
</div>
`;

exports[`Avatar name renders properly 1`] = `
<div
  className="theme-portal"
>
  <div
    className="profileAvatar"
  >
    <span>
      KB
    </span>
  </div>
</div>
`;

exports[`Avatar with photo renders properly 1`] = `
<div
  className="theme-portal"
>
  <div
    className="profileAvatar"
  >
    <img
      alt="Lionel Messi"
      src="https://e00-marca.uecdn.es/assets/multimedia/imagenes/2018/08/19/15347041965884.jpg"
    />
  </div>
</div>
`;
